### ISSUE TITLE
Add option to enable/disable logging the ffmpeg command on sendMp4AsSticker

### ISSUE THREAD

**Issue by WaxArsatia** [open-wa/wa-automate-nodejs#1250]
https://docs.openwa.dev/classes/client.html#sendmp4assticker

Every run its will log
```
Started ffmpeg -f mp4 -i pipe:0 -y -vcodec libwebp -vf crop=w='min(min(iw,ih),500)':h='min(min(iw,ih),500)',scale=500:500,setsar=1,fps=10 -loop 0 -ss 00:00:00.0 -t 00:00:05.0 -preset default -an -vsync 0 -s 512:512 -f webp C:\Users\Administrator\AppData\Local\Temp\processing.2d9kwpmorl.webp
```
This is solution to hide it, but how can im add the parameter to hide ffmpeg command?
https://superuser.com/questions/326629/how-can-i-make-ffmpeg-be-quieter-less-verbose

Its should be nice if next update can choose to enable or disable the output command log

**Comment by smashah** [comment]
To be honest the logging was set up there to make sure it's working properly. I'm just going to turn it off entirely.

**Comment by smashah** [comment]
@Waxaranai in the next version the FFmpeg logs will be off by default. In order to turn them back on you can do so by setting `processOptions.log` to `true`.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.14](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.13...3.1.14) (2021-01-19)

- build(deps): bump puppeteer-extra-plugin-devtools from 2.2.14 to 2.2.15 [`#1235`](https://github.com/open-wa/wa-automate-nodejs/pull/1235)
- build(deps): bump puppeteer-extra-plugin-block-resources from 2.2.7 to 2.2.8 [`#1240`](https://github.com/open-wa/wa-automate-nodejs/pull/1240)
- build(deps): bump open from 7.3.0 to 7.3.1 [`#1242`](https://github.com/open-wa/wa-automate-nodejs/pull/1242)
- build(deps-dev): bump @types/line-reader from 0.0.30 to 0.0.32 [`#1239`](https://github.com/open-wa/wa-automate-nodejs/pull/1239)
- build(deps): bump puppeteer-extra-plugin-stealth from 2.6.5 to 2.6.6 [`#1241`](https://github.com/open-wa/wa-automate-nodejs/pull/1241)
- build(deps): bump swagger-ui-dist from 3.38.0 to 3.40.0 [`#1246`](https://github.com/open-wa/wa-automate-nodejs/pull/1246)
- build(deps): bump puppeteer-extra from 3.1.15 to 3.1.16 [`#1243`](https://github.com/open-wa/wa-automate-nodejs/pull/1243)
- build(deps): bump socket.io from 3.0.4 to 3.1.0 [`#1247`](https://github.com/open-wa/wa-automate-nodejs/pull/1247)
- build(deps-dev): bump husky from 4.3.6 to 4.3.8 [`#1255`](https://github.com/open-wa/wa-automate-nodejs/pull/1255)
- Release 3.1.14 [`19da707`](https://github.com/open-wa/wa-automate-nodejs/commit/19da70734fd15b98a77f785cd83041ba289eea26)
- fix: ffmpeg logs now off by default #1250 [`448d3bf`](https://github.com/open-wa/wa-automate-nodejs/commit/448d3bf5ea5a85a08a6ae4f04788c61492ecd2c2)
- updated package lock [`38623ea`](https://github.com/open-wa/wa-automate-nodejs/commit/38623ead6192405980132577f13679ee931e119d)
- Feature: `isGroupIdUnsafe` [insiders] #1114 [`5b52df9`](https://github.com/open-wa/wa-automate-nodejs/commit/5b52df94e68dcf84250d8e949c7259c1f6637316)
- patch: better group function debugging [`e1dff0b`](https://github.com/open-wa/wa-automate-nodejs/commit/e1dff0b62e9beed8fc7901a7de9cea7f8029c99f)
